php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -1398,6 +1398,22 @@ class NDB_BVL_Instrument extends NDB_Page
         return false;
     }
 
+    /**
+     * Gets the VisitLabel for the timepoint to which this instrument pertains
+     *
+     * @return String for the VisitLabel
+     */
+    function getVisitLabel{
```

Comment:
this isn't syntactically valid PHP. You need the argument list.

(You should also run phpcs on it or Travis is going to fail. You can just run `npm run lint:php` as a shortcut to run PHPCS and `php -l`.)